test/gtest/stringify.cpp

```diff
@@ -100,17 +102,29 @@ adding unique symbol for Try Body Start
 in visitExpression for nop
 adding unique symbol for End
 adding unique symbol for Try Catch Start
-in visitExpression for i32.const 8
-in visitExpression for drop
+in visitExpression for block
 adding unique symbol for End
 adding unique symbol for Try Catch Start
-in visitExpression for i32.const 15
-in visitExpression for drop
+in visitExpression for block
 adding unique symbol for End
 adding unique symbol for Try Body Start
 in visitExpression for nop
 adding unique symbol for End
 adding unique symbol for Try Catch Start
+in visitExpression for block
+adding unique symbol for End
+adding unique symbol for Block Start
+in visitExpression for pop i32
+in visitExpression for i32.const 8
+in visitExpression for drop
+adding unique symbol for End
+adding unique symbol for Block Start
+in visitExpression for pop i32
+in visitExpression for i32.const 15
+in visitExpression for drop
+adding unique symbol for End
+adding unique symbol for Block Start
+in visitExpression for pop i32
```

Comment:
Why is there such a big diff here?

Reply:
The extra two blocks mean that the top-level walker loop has two more control flow structures to pop off the control structure queue, generating more logs. The new parser also inserts a couple pops that are missing in the input that the old parser did not generate.

Reply:
Which two new blocks? Sorry, I'm missing something here.

Reply:
Ah, those would be the blocks the new parser generates for the `catch` clauses in the input. The input does not include the implicit `pop` instruction at the beginning of the catch clauses, so the old parser just used `(drop ...)` as their bodies. Since the new parser inserts the `pop` instruction, the catch clauses have multiple top-level instructions, so they are wrapped in a block.

(And if something seems fishy about this, it may help to know that the input program would not pass validation)

Reply:
Thanks, now I see!